src/Umbraco.Web/Editors/MemberController.cs

```diff
@@ -96,16 +82,18 @@ protected MembershipScenario MembershipScenario
 
             if (MembershipScenario == MembershipScenario.NativeUmbraco)
             {
-                long totalRecords;
                 var members = Services.MemberService
-                    .GetAll((pageNumber - 1), pageSize, out totalRecords, orderBy, orderDirection, orderBySystemField, memberTypeAlias, filter).ToArray();
+                    .GetAll((pageNumber - 1), pageSize, out var totalRecords, orderBy, orderDirection, orderBySystemField, memberTypeAlias, filter).ToArray();
                 if (totalRecords == 0)
                 {
                     return new PagedResult<MemberBasic>(0, 0, 0);
                 }
-                var pagedResult = new PagedResult<MemberBasic>(totalRecords, pageNumber, pageSize);
-                pagedResult.Items = members
-                    .Select(Mapper.Map<IMember, MemberBasic>);
+
+                var pagedResult = new PagedResult<MemberBasic>(totalRecords, pageNumber, pageSize)
+                {
+                    Items = members
+                        .Select(Mapper.Map<IMember, MemberBasic>)

```

Comment:
For this to work properly, you need to use `AutoMapperExtensions.MapWithUmbracoContext` instead of `Mapper.Map`, the reason is because we want to inject the current `UmbracoContext` into the mapping operation, this is why in the mapper that you created you do `ResolveCore(source.Context.GetUmbracoContext()`. It is working without this operation because that will revert back to using the `UmbracoContext.Current` singleton, however we don't want to use that because singletons should be avoided since it leads to un-testable code and spaghetti code.